src/raster/r.convergence/r.convergence.html

```diff
@@ -1,51 +1,80 @@
 <h2>OPTIONS</h2>
 <dl>
 <dt><b>-s</b></dt>
-<dd>Increase convergence if slope value is high. Slope parameter radically slow down computation time, especially if window parameter is high. If slope is used addational modifier is used according to formula: sin(current)*sin(target) + cos(current)*cos(target). if slope of current and target cells are equal. The modifier is 1. If not, the modifier is applied with formula: acos(cos(convergence) * modifier)  </dd>
+<dd>Increase convergence if slope value is high. 
+Slope parameter radically slows down computation time, 
+especially if the window parameter is high. 
+If slope is used, a slope modifier is used according to the formula: 
+sin(current)*sin(target) + cos(current)*cos(target). 
+If the slope of current and target cells are equal, 
+this modifier's value will be 1. 
+The modifier is applied with the formula: 
+acos(cos(convergence) * modifier)
+</dd>
+
 <dt><b>-c</b></dt>
-<dd>use circular window instead of suqare (default)</dd>
+<dd>Use circular window instead of square (default)</dd>
 
 <dt><b>input</b></dt>
-<dd>Digital elevation model. Data can be of any type and any projection. To calculate relief convergnece, r.convergence uses real distance which is recalculated into cell distance, according formula: <br><code>distance_between_current_cell_and_traget_cell/distance_between_current_cell_and_nearest_neighbour_cell.</code> It is important if convergence is calculated for large areas in LatLong projecton.
+<dd>Digital elevation model. Data can be of any type and any projection. 
+To calculate relief convergence, r.convergence uses real distances 
```

Comment:
```suggestion
To calculate relief convergence, <em>r.convergence</em> uses real distances 
```

Minor update following https://trac.osgeo.org/grass/wiki/Submitting/Docs#Markupstyleguide